index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    
+
+  <div id="main">
+      <h1 id="title"> Ezio the great assassin</h1>
+   <div id="img-div">
+    <a id="tribute-link" target="_blank" href="https://www.youtube.com/watch?v=0P3d7-mHCbg">
+     
```

Comment:
Great job on explaining how to take the link from youtube but I think you can also explain why you suggested that so instead of saying:
   > Good job so far but I think you need to change something in this section,

You can say:

> Good job so far but I think you need to change something in this section, because the video is not working well so it would be better if you go to youtube and take the embedded link from the video to work as you like.👌🏻👌🏻